http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/package.scala

```diff
@@ -32,6 +32,12 @@ package object http {
       .compile
       .lastOrError
 
+  def readFileToBytes[F[_]: Sync: ContextShift](path: Path, blocker: Blocker): F[List[Byte]] =
```

Comment:
where is this used? also why do we need to convert `Stream[F, Byte]` into `F[List[Byte]]`?

Reply:
its used here https://github.com/DataBiosphere/leonardo/pull/1511/files#diff-d08c4643745ad6f27c31469d39980e33R94 because we require a java Array eventually to pass to a client lib, and don't actually consume the bytes in Leo. It may be possible to transform to stream into this though... I originally made this because the stream wasn't working, but it turned out to be an unrelated issue

Reply:
I think you can do `.compile.to(Array)` on a `Stream[F, Byte]` directly to get an Array of bytes